containers.sh

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC2034,SC2143
+set -euo pipefail
+
+# Network array - to add more, separate by space and double qoutes e.g. "todo-app" "your-second-network"
+networks=("todo-app")
+# Volume array - to add more, separate by space and double qoutes, e.g. "todo-mysql-data" "your-second-volume"
+volumes=("todo-mysql-data")
+# Path to docker images
+images="${PWD}/docker/images"
+# Set app container name
+app="todo-app"
+# Set db container name
+db="todo-db"
+
+log(){
+  echo "$(date +'%T') $1"
+}
+
+networks(){
+  docker network create \
+    "${network}"
+}
+
+volumes(){
+  docker volume create \
+    "${volume}"
+}
+
+todo_app(){
+  docker run \
+    --detach \
+    --name "${app}" \
+    --publish 3000:3000 \
+    --workdir "/app" \
+    --volume "${images}/getting-started/app:/app" \
```

Comment:
Ah okay, I didn't realise it could be interpreted that way. I honestly just wanted 'good names' without too many underscores etc., and thought I wrote my way out of it, by making good comments :D